src/reducers/dataEntryErrorsReducer.js

```diff
@@ -11,7 +21,16 @@ const locationErrorsReducer = (state = [], action) => {
       if (!action.payload || !action.payload.error) {
         return state;
       }
-
+      const extra = {
+        response: {
+          payload: action.payload.error.response.data,
+          ...action.payload.error.response,
+        }
+      }
+      Sentry.captureException(action.payload.error, {
```

Comment:
Would it make sense to use Sentry's SDK for [React](https://docs.sentry.io/platforms/javascript/guides/react/), which includes out-of-the-box [Redux support](https://docs.sentry.io/platforms/javascript/guides/react/features/redux/)?

Haven't used it before, so don't know if it would fully avoid the need to explicitly set it up for individual error reducers like this, but if so, it might give more comprehensive coverage with less code.

Reply:
This is a good idea, but let's punt it. For now, we just want to track these HTTP errors.

Reply:
Sounds good.